examples/pie_and_polar_charts/pie_features.py

```diff
@@ -1,41 +1,103 @@
 """
-===============
-Basic pie chart
-===============
+==========
+Pie charts
+==========
 
-Demo of a basic pie chart plus a few additional features.
-
-In addition to the basic pie chart, this demo shows a few optional features:
-
-* slice labels
-* auto-labeling the percentage
-* offsetting a slice with "explode"
-* drop-shadow
-* custom start angle
+Demo of plotting a `~matplotlib.axes.Axes.pie` chart
```

Comment:
I'm guessing this has been discussed elsewhere, and out of scope for this PR, but just wanted to bring up that having an intersphinx link like this in the first line doesn't look great on the thumbnail:
<img width="231" alt="Screenshot 2023-01-12 at 21 27 42" src="https://user-images.githubusercontent.com/6197628/212185025-31f00f2b-0b49-4d52-a9d2-8b3e800c3b84.png">

Reply:
Yes, this should be avoided.

Reply:
Though note, I think it's a regression in sphinx-gallery that they stopped reducing the thumbnail from rst back to plain text. I guess there isn't a fix out yet. https://github.com/sphinx-gallery/sphinx-gallery/issues/1002